content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,245 @@
+  }}
+  loop={true}
+  title="Multi-version upgrades"
+  caption="Upgrade to 4.0 directly from version 3.20 or higher with multi-version upgrades"
+/>
+
+<a href="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<Badge link="https://docs.sourcegraph.com/admin/analytics" text="Admin" color="violet" size="small" />
+
+#### Understand usage and quantify the value of using Sourcegraph with in-product analytics
+[In-product analytics](https://docs.sourcegraph.com/admin/analytics) helps admins understand user engagement, measure efficiency in terms of time saved, and calculate the value of utilizing Sourcegraph.
+
+With charts that detail usage over time and value calculators that quantify the amount of time saved as users search, create insights, run batch changes, and more, you can now report on:
+- The number of searches developers have run in Sourcegraph, and how much development time has been saved as a result.
+- How many Code Insights charts have been created, viewed, and interacted with.
+- The number of changesets that have been created and merged via Batch Changes, and how much time has been saved as a result.
+- How frequently code navigation has been used by developers to more quickly understand code.
+- How often IDE and browser extensions are used across your organization, and how much time has been saved as a result.
+- The number of navigation events and the percentage of your repositories that allow precise code navigation.
+
+The feature value calculators use data derived from event logs and the default time-saved values can be customized to reflect how your team uses Sourcegraph.
+
+<div className="tw-mb-sm">
+  <Figure
+    src="https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Search.png"
+    alt="In-product analytics"
+    caption="Understand usage and quantify the value of using Sourcegraph with in-product analytics"
+  />
+</div>
+
+<a href="https://docs.sourcegraph.com/admin/analytics" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<Badge link="https://docs.sourcegraph.com/admin/observability/opentelemetry" text="Admin" color="violet" size="small" />
+
+#### Easily export traces using OpenTelemetry
+
+[OpenTelemetry (OTEL)](https://opentelemetry.io/) is an open specification for exporting observability data into a user’s preferred systems. We aim to make OpenTelemetry the default way to quickly and easily export observability data from Sourcgraph, starting with tracing in Sourcegraph 4.0. This is a breaking change as we now require you to export tracing with the [OpenTelemetry collector](https://opentelemetry.io/docs/collector/). This is deployed by default in all Sourcegraph deployment methods and replaces the Jaeger agents and collectors that was previously the default. 
```

Comment:
```suggestion
We’ve added [OpenTelemetry (OTEL)](https://opentelemetry.io/) support in Sourcegraph 4.0 so you can easily export tracing data from Sourcegraph into your preferred observability solution. This is a breaking change as we now require you to export tracing with the [OpenTelemetry collector](https://opentelemetry.io/docs/collector/). This is deployed by default in all Sourcegraph deployment methods and replaces the Jaeger agents and collectors that was previously the default. 
```

@lcolston08 @quinnhare - is this accurate - I am not sure how to talk about "tracing data?" I don't know that we need to explain what OpenTelemetry is. If we want to keep the sentence, then I'd recommend:
[OpenTelemetry (OTEL)](https://opentelemetry.io/) is an open specification for exporting observability data. We’ve added OpenTelemetry support in Sourcegraph 4.0 so you can easily export tracing data from Sourcegraph into your preferred observability solution This is a breaking change as we now require you to export tracing with the [OpenTelemetry collector](https://opentelemetry.io/docs/collector/). This is deployed by default in all Sourcegraph deployment methods and replaces the Jaeger agents and collectors that was previously the default.

Reply:
I personally think we should explain what OpenTelemetry is since it's not a feature that we own.